api_examples/sequential_learning_api_tutorial.ipynb

```diff
@@ -0,0 +1,692 @@
+    "\n",
+    "    dv_id = client.data_views.create(\n",
+    "        dv_config, \n",
+    "        name=view_name, \n",
+    "        description=view_desc\n",
+    "    )\n",
+    "    return dv_id"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def run_design(client:CitrinationClient, view_id:int, dataset_id:int,\n",
+    "                        num_candidates_per_iter:int,\n",
+    "                        design_effort:int, wait_time:int, \n",
+    "                        num_sl_iterations:int, input_properties:List[str],\n",
+    "                        target:List[str], print_output:bool,\n",
+    "                        true_function:Callable[[np.ndarray], float],\n",
+    "                        score_type:str,\n",
+    "                        ) -> Tuple[List[float], List[float]]:\n",
+    "    '''Runs SL design\n",
+    "    \n",
+    "    :param client: Client object\n",
+    "    :type client: CitrinationClient\n",
+    "    :param view_id: View ID\n",
+    "    :type view_id: int\n",
+    "    :param dataset_id: Dataset ID\n",
+    "    :type dataset_id: int\n",
+    "    :param num_candidates_per_iter: Candidates in a batch\n",
+    "    :type num_candidates_per_iter: int\n",
+    "    :param design_effort: Effort from 1-30\n",
+    "    :type design_effort: int\n",
+    "    :param wait_time: Wait time in seconds before polling API\n",
+    "    :type wait_time: int\n",
+    "    :param num_sl_iterations: SL iterations to run\n",
+    "    :type num_sl_iterations: int\n",
+    "    :param input_properties: Inputs\n",
+    "    :type input_properties: List[str]\n",
+    "    :param target: (\"Output property\", {\"Min\", \"Max\"})\n",
+    "    :type target: List[str]\n",
+    "    :param print_output: Whether or not to print outputs\n",
+    "    :type print_output: bool\n",
+    "    :param true_function: Actual function for evaluating measured/true values\n",
+    "    :type true_function: Callable[[np.ndarray], float]\n",
+    "    :param score_type: MLI or MEI\n",
+    "    :type score_type: str\n",
+    "    :return: 2-tuple: list of predicted scores/uncertainties; list of measured scores/uncertainties\n",
+    "    :rtype: Tuple[List[float], List[float]]\n",
+    "    '''\n",
+    "\n",
+    "\n",
+    "\n",
+    "    best_sl_pred_vals = []\n",
+    "    best_sl_measured_vals = []\n",
+    "\n",
+    "    _wait_on_ingest(client, dataset_id, wait_time, print_output)\n",
+    "\n",
+    "    for i in range(num_sl_iterations):\n",
+    "        if print_output:\n",
+    "            print(f\"\\n---STARTING SL ITERATION #{i+1}---\")\n",
+    "\n",
+    "        _wait_on_ingest(client, dataset_id, wait_time, print_output)\n",
+    "        _wait_on_data_view(client, dataset_id, view_id, wait_time, print_output)\n",
+    "            \n",
+    "        # Submit a design run\n",
+    "        design_id = client.submit_design_run(\n",
```

Comment:
hmm this needs Admin privileges right? Not urgent for this demo, but how will we let external people do this step? Will have to check in with Kevin/Travis/Sean.